packages/brick-kit/src/internal/bindListeners.ts

```diff
@@ -519,13 +539,30 @@ function builtinStateListenerFactory(
       return;
     }
     const tplContext = getTplContext(context.tplContextId);
-    const [name, value] = argsFactory(args, context, event);
-    tplContext.state.updateValue(
-      name as string,
-      value,
-      method === "update" ? "replace" : method,
-      callback
-    );
+    const isBatch = Array.isArray(args) && args.every(isObject);
+    const computedMethod = method === "update" ? "replace" : method;
+    if (isBatch) {
+      const computedArgs = argsFactory(args, context, event) as Array<{
+        name: string;
+        value: unknown;
+      }>;
+      if (batch) {
```

Comment:
只有同步执行的 `replace` 可以进行 batch 操作，其他的 `refresh` 和 `load` 是异步的，不符合我们最初设计的场景

```suggestion
      if (batch && computedMethod === "replace") {
```